src/test/java/core/basesyntax/strategy/handlers/BalanceHandlerTest.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.strategy.handlers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class BalanceHandlerTest {
+    private OperationHandler operationHandler;
+
+    @BeforeEach
+    void setUp() {
+        operationHandler = new BalanceHandler();
+    }
+
+    @Test
+    void testOperateBalance_ok() {
```

Comment:
what if someone would try to use this method with null key? or assign negative value?